addon/components/frost-fixed-table.js

```diff
@@ -334,5 +338,20 @@ export default Component.extend({
   // == Actions ===============================================================
 
   actions: {
+    /**
+     * Generic action dispatcher, so that cell renderers can indicate arbitrary events.
+     * Your handler is then responsible for doing stuff based on actions like 'click' or 'input'.
+     *
+     * @param {Number} row - data rows are zero based, and header has row -1
+     * @param {Number} col - column index
+     * @param {String} action - this comes after row/col as Ember lets us include those in action closures easily.
+     * @param {Object[]} args - any additional data
+     */
+    genericDispatch (row, col, action, args) {
```

Comment:
I guess if it's just at the top level it's not such a big deal. Personally, I don't see what's wrong with `row`, `col`, `action`, `args` for the consumer though.